KNOWN_ISSUES.md

```diff
@@ -0,0 +1,117 @@
+# Known issues
+
```

Comment:
what do you think about adding a TOC here?

Reply:
I think this is good idea for both KNOWN_ISSUES.md and FAQ.md
To maintain TOC up to date we should have `update` and `validate` scripts, so I would propose to do this as a follow up

Reply:
sgtm :+1: